### PR TITLE
Prettify chat loading view

### PR DESCRIPTION
**Pull request by siepra**
This PR prettifies the loading component displayed while waiting for the chat screen to become ready. It's based on the following design https://www.figma.com/file/y8h6w8PYR9jyI3zjYHL9Cl/Mobile-%2B-desktop-%2B-prototypes?type=design&node-id=1427-34765&mode=design&t=hh3MmRfPqqyPM7xk-4.

What's most important it temporarily **hides the message's input** which doesn't work until the screen is fully loaded. That might have been a reason of recent rejection from Google Play.

### Pull Request Checklist

- [x] I have linked this PR to related GitHub issue.
- [x] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-[2.0.0-alpha.11]
+[2.0.0-alpha.14]
```
Comment (EmiM) [inline CHANGELOG.md:1]: Why did you change .11 to .14? 
And shouldn't the new info be in [unreleased] section above?

**Review by vinkabuki — APPROVED** [review]

**Review by EmiM — APPROVED** [review]
